app/assets/src/components/views/PipelineViz.jsx

```diff
@@ -220,12 +229,46 @@ class PipelineViz extends React.Component {
     return stepClassName.replace(/^(PipelineStep(Run|Generate)?)/, "");
   }
 
-  handleStepClick(stageIndex, info) {
-    const clickedNodeId = info.nodes[0];
+  inverseTransformDOMCoordinates(x, y) {
+    const panZoomContainerDOM = ReactDOM.findDOMNode(this.panZoomContainer)
```

Comment:
would it be possible to do this with a React ref instead?

Reply:
I don't think it's possible since it's trying to get the styling of the first child of the ref (instead of the ref itself)